### ISSUE TITLE
Doesn't work with Materialize UI

### ISSUE THREAD

**Issue by sameer-coder** [LeaVerou/awesomplete#16718]
The materialize input box stop showing its default animation when clicked on a input box although autocomplete works.

``` html
<input id="demo" type="text" class="validate tooltipped awesomplete" data-position="bottom" data-delay="10" data-tooltip="Demo">
```

``` javascript
    var demo_input = document.getElementById("demo"); 
    new Awesomplete(demo_input, {
        list: ["W141001900", "W141001901", "W141001903", "W131001229" "W111001089"],
        minChars: 3,
        maxItems: 20
    });
```

**Comment by LeaVerou** [comment]
What animation was that? Is there a live demo of this anywhere?

**Comment by Garbee** [comment]
Seems to be talking about the [materialize css framework](http://materializecss.com/).  Not sure which animation is being discussed specifically though.

**Comment by sameer-coder** [comment]
Yes the materialize framework. Its the input box placeholder text which gets animated when you click on it. but the minute you add the awesomplete class to the input that stops working. You can check the animation here http://materializecss.com/forms.html when you click on the last name input there is some animation there.

Got a jsfiddle for you here
https://jsfiddle.net/06Lt3vm9/2/

Try clicking on the last name without modifying the code. Then add the awesomplete class to the last name input and then see. Awesomplete works fine but the animation stops working

**Comment by erquhart** [comment]
This is happening because Materialize input/label animations require the two elements to be siblings, and awesomplete (like most (all?) autocomplete libs) wraps the target input in a template, making the input one DOM level lower than the label.

I'm actually using Materialize for a Chrome extension and am up against the same issue. The best concept I've mustered is using a class along the lines of `awesomplete-include` to indicate irrelevant siblings that need to be wrapped along with the input. Gets a tad complicated when you consider that multiple awesomplete instances may be running under the same DOM node, but this feature could simply require that target inputs not share a node with other target inputs.

I'll submit a PR if I crack it.

**Comment by clemsos** [comment]
Looks like it works well with Materialize for me. See code here http://codepen.io/clemsos/pen/pbLEJJ
